core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -638,6 +638,15 @@ cairo.sql.copy.root=import
 # When disabled, SQL executed by materialized view refresh job always runs single-threaded.
 #cairo.mat.view.parallel.sql.enabled=true
 
+# Defines the desired number of base table rows to be scanned by queries run by the materialized view refresh job.
+#cairo.mat.view.rows.per.query.estimate=10000000
+
+# Minimal time interval used by the refresh job when running materialized view queries.
+#cairo.mat.view.min.refresh.interval=1m
+
+# Maximum number of rows inserted by the refresh job in a single transaction.
+#cairo.mat.view.insert.as.select.batch.size=1000000
```

Comment:
undocumented cairo.mat.view.max.refresh.retries.on.oom

Reply:
Removed this property as per your other comment.

Reply:
It looks like now we retry up to the max number of recompilation attempts which sounds weird. We should at least rename the `getMatViewMaxRecompileAttempts` property and change its documentation.